### DISCUSSION TITLE
Frontmatter的features中的文字怎么实现两端对齐？

### DISCUSSION THREAD

**Discussion by hyfly233** [vuepress/core#1326] [Q&A] [answered]
在 [features](https://v2.vuepress.vuejs.org/zh/reference/default-theme/frontmatter.html#features) 中，features 中的文字该怎么实现文字的两端对齐？

**Comment by Mister-Hope** [comment, marked as answer]
Add your own css to customize styles.

**Reply by hyfly233** [reply]
Thanks，i found the answer on this page [styles](https://v2.vuepress.vuejs.org/zh/reference/default-theme/styles.html)

I wrote the following in the scss file (.vuepress/styles/index.scss) to solve this problem
```scss
.home .feature p {
  text-align: justify !important;
}
```
